playbooks/roles/vault/tasks/main.yml

```diff
@@ -38,7 +38,7 @@
   notify: restart vault
 
 - name: Determine current Vault version
-  shell: vault version | sed -n 's/Vault v\(.*\)/\1/p'
+  shell: vault version | cut -d" " -f 2 - | cut -b 2- -
```

Comment:
Misc fix - I was doing it wrong in a previous PR, and this fixes it. Used same tactic for consul-template version detection, so decided to utilize it here.